providers/fab/docs/changelog.rst

```diff
@@ -47,144 +46,29 @@ Breaking changes
 
 * ``Prepare FAB provider to set next version as major version (#43939)``
 * ``Remove deprecations from fab provider (#44198)``
+* ``Rename 'get_permitted_dag_ids' and 'filter_permitted_dag_ids' to 'get_authorized_dag_ids' and 'filter_authorized_dag_ids' (#47640)``
+* ``Set simple auth manager as default (#47691)``
 
 Features
 ~~~~~~~~
 
 * ``Set up JWT token authentication in Fast APIs (#42634)``
 * ``AIP-79 Support Airflow 2.x plugins in fast api. Embed a minimal version of the Flask application in fastapi application (#44464)``
+* ``AIP 84 - Add auth for asset alias (#47241)``
+* ``AIP-81 | AIP-84 | Include Token Generation Endpoints in FAB (#47043)``
+* ``AIP-84 Add Auth for DAG Versioning (#47553)``
+* ``AIP-84 Add Auth for backfill (#47482)``
 
 Misc
 ~~~~
 
 * ``AIP-81 Move CLI Commands to directories according to Hybrid, Local and Remote (#44538)``
```

Comment:
Small note these changes have been reverted in here  ``AIP-81: Flatten core CLI commands (#48224)`` and moved to airflowctl

Reply:
Moved under the ignore section